system/lib/webgpu/webgpu.cpp

```diff
@@ -27,3 +38,24 @@ WGPUInstance wgpuCreateInstance(const WGPUInstanceDescriptor* descriptor) {
 
 void wgpuInstanceReference(WGPUInstance) { /* no-op for now */ }
 void wgpuInstanceRelease(WGPUInstance) { /* no-op for now */ }
+
+// WGPUSurface
+
+void wgpuSurfaceGetCapabilities(WGPUSurface surface,
+                                WGPUAdapter adapter,
+                                WGPUSurfaceCapabilities* capabilities) {
```

Comment:
Assert `capabilities->nextInChain == nullptr` because we don't support any extensions and GetCapabilities doesn't have a return code yet (it will, later).
```suggestion
                                WGPUSurfaceCapabilities* capabilities) {
  assert(capabilities->nextInChain == nullptr); // TODO: Return WGPUStatus_Error
```